astronomer/providers/sftp/hooks/sftp.py

```diff
@@ -0,0 +1,127 @@
+from fnmatch import fnmatch
+from typing import List
+
+import asyncssh
+from airflow.exceptions import AirflowException
+from airflow.hooks.base import BaseHook
+from asgiref.sync import sync_to_async
+
+
+class SFTPHookAsync(BaseHook):
+    """
+    Interact with an SFTP server via asyncssh package
+
+    :param sftp_conn_id: SFTP connection ID to be used for connecting to SFTP server
+    :param host: hostname of the SFTP server
+    :param port: port of the SFTP server
+    :param username: username used when authenticating to the SFTP server
+    :param password: password used when authenticating to the SFTP server
+                     Can be left blank if using a key file
+    :param known_hosts: path to the known_hosts file on the local file system
+                     If known_hosts is set to the literal "none", then no host verification is performed
+    :param key_file: path to the client key file used for authentication to SFTP server
+    :param passphrase: passphrase used with the key_file for authentication to SFTP server
+    """
+
+    conn_name_attr = "ssh_conn_id"
+    default_conn_name = "sftp_default"
+    conn_type = "sftp"
+    hook_name = "SFTP"
+    default_known_hosts = '"~/.ssh/known_hosts"'
+
+    def __init__(  # nosec: B107
```

Comment:
why do we need this `# nosec: B107`?

Reply:
Static checker thinks that a password is exposed with:
```
password: str = "",
```